docker-compose.kibana.yaml

```diff
@@ -1,5 +1,5 @@
 #ddev-generated
-version: '3.6'
+version: '3'
```

Comment:
No, the version tag is completely obsolete and should be removed.

Reply:
sure i remove :)